index.js

```diff
@@ -54,6 +65,21 @@ module.exports = function (src, dest, opts) {
 		return new Promise(function pipeToDest(resolve, reject) {
 			var write = fs.createWriteStream(dest, {flags: opts.overwrite ? 'w' : 'wx'});
 
+			read.on('data', function () {
+				if (size === 0) {
+					return;
+				}
+				var written = write.bytesWritten;
+				var percent = written / size;
+				progressEmitter.emit('progress', {
+					src: src,
+					dest: dest,
```

Comment:
Not entirely sure what you're saying, but I didn't imply they should be removed.